projects/fsastorefrontstyles/scss/overrides/update-email.scss

```diff
@@ -1,11 +1,26 @@
 %cx-update-email {
```

Comment:
Since this is their component and they changed the structure of it, the only way to fix styles is by adding more CSS properties.